go.mod

```diff
@@ -5,17 +5,14 @@ go 1.12
 require (
 	github.com/gobuffalo/flect v0.1.5
 	github.com/golang/protobuf v1.3.1 // indirect
-	github.com/inconshreveable/mousetrap v1.0.0 // indirect
-	github.com/kr/pretty v0.1.0 // indirect
 	github.com/onsi/ginkgo v1.8.0
 	github.com/onsi/gomega v1.5.0
 	github.com/spf13/afero v1.2.2
 	github.com/spf13/cobra v0.0.3
 	github.com/spf13/pflag v1.0.3
 	golang.org/x/net v0.0.0-20190620200207-3b0461eec859 // indirect
 	golang.org/x/sys v0.0.0-20190621203818-d432491b9138 // indirect
-	golang.org/x/text v0.3.2 // indirect
 	golang.org/x/tools v0.0.0-20190614205625-5aca471b1d59
-	gopkg.in/check.v1 v1.0.0-20180628173108-788fd7840127 // indirect
 	gopkg.in/yaml.v2 v2.2.2
+	sigs.k8s.io/controller-tools v0.2.0-beta.4 // indirect
```

Comment:
I believe this is a side-effect when you install CT through go get.
`go mod tidy` should clean it up for you.

Reply:
Fixed that.